content/faq/graph/internet-explorer.adoc

```diff
@@ -0,0 +1,9 @@
+---
+title: Does Kiali support Internet Explorer?
```

Comment:
Seems like this should be in "general", not "graph".

Reply:
True, will update and merge